variables.tf

```diff
@@ -365,6 +365,10 @@ variable "firelens_configuration" {
     options = map(string)
   })
   default = null
+variable "ecs_task_execution_iam_role_arn" {
```

Comment:
you're missing a closing `}` above.

lets ask the user for the existing role name (rather than the ARN) and then add the following:

```hcl
data "aws_iam_role" "existing" {
	count = var.ecs_task_role_name != "" ? 1 : 0
	name =  var.ecs_task_role_name
}
```

then with our outputs we should be able to do something like:

```hcl
output "task_role_id" {
  description = "The Atlantis ECS task role id"
  value       = var.ecs_task_role_name == "" ? aws_iam_role.ecs_task_execution[0].id : data.aws_iam_role.existing[0].id
}
```